var/spack/repos/builtin/packages/mpich/package.py

```diff
@@ -19,9 +19,9 @@ class Mpich(AutotoolsPackage):
     git      = "https://github.com/pmodels/mpich.git"
     list_url = "https://www.mpich.org/static/downloads/"
     list_depth = 1
-
+ 
```

Comment:
Whitespace error

Reply:
all style checks have been addressed and I checked the latest CI check and i saw no errors.